### PR TITLE
#26: Error when using the library with Yarn's PnP

### PR DESCRIPTION
**Pull request by cipivanov**
See https://github.com/abhinaba-ghosh/axe-playwright/issues/19 for context. 

Now we will check if we are running in a PnP environment and load the needed axe-core file slightly differently.

### CODE REVIEW

**Review by merceyz — CHANGES_REQUESTED** [review]
This is not the way to go about fixing this, using a simple `require.resolve('axe-core/axe.min.js')` to get the path is the correct solution

**Comment by abhinaba-ghosh** [comment]
I second the opinion

**Comment by cipivanov** [comment]
@merceyz thanks for the suggestion, should be fixed now.

File: src/index.ts [hunk 1]
```diff
@@ -19,11 +19,11 @@ declare global {
...
-    'node_modules/axe-core/axe.min.js',
-    'utf8',
-  )
-  await page.evaluate((axe: string) => window.eval(axe), axe)
-}
+    `${require.resolve("axe-core/axe.min.js")}`,
+    "utf8"
+  );
+  await page.evaluate((axe: string) => window.eval(axe), axe);
+};
```
Comment (merceyz) [inline src/index.ts:26]:
```suggestion
    require.resolve('axe-core/axe.min.js'),
    'utf8',
  )
  await page.evaluate((axe: string) => window.eval(axe), axe)
}
```

Comment (cipivanov) [reply src/index.ts]: I think this will fail the codacy job. Probably needs a bit more tuning.

**Pull request merged**